package.json

```diff
@@ -18,27 +18,34 @@
   "scripts": {
     "blc:suspensive.org": "pnpm blc https://suspensive.org --ro",
     "build": "turbo run build",
-    "build:watch": "turbo run build:watch --parallel",
+    "build:watch": "turbo watch build",
     "changeset": "changeset",
     "changeset:publish": "pnpm prepack && changeset publish",
     "changeset:version": "changeset version && pnpm i --lockfile-only",
     "ci:all": "pnpm ci:attw && pnpm ci:eslint && pnpm ci:publint && pnpm ci:sherif && pnpm ci:type && pnpm ci:test && pnpm ci:knip && pnpm ci:bench && pnpm build",
     "ci:attw": "turbo run ci:attw",
+    "ci:attw:watch": "turbo watch ci:attw",
     "ci:bench": "turbo run ci:bench",
-    "ci:eslint": "turbo run ci:eslint",
+    "ci:eslint": "turbo watch ci:eslint",
+    "ci:eslint:watch": "turbo watch ci:eslint",
     "ci:knip": "pnpm prepack && pnpm knip",
     "ci:publint": "turbo run ci:publint",
+    "ci:publint:watch": "turbo watch ci:publint",
     "ci:sherif": "sherif --ignore-package \"./examples/*\" --ignore-package \"./websites/*\" --ignore-package \"./docs/*\" --ignore-dependency \"@tanstack/react-query\" ",
     "ci:test": "turbo run ci:test",
-    "ci:test:watch": "turbo run ci:test:watch --parallel",
+    "ci:test:watch": "turbo watch ci:test",
+    "ci:test:watch:ui": "turbo run ci:test:watch:ui --parallel",
```

Comment:
I named it ci:test:watch:ui because, unlike ci:test:watch which operates in turbo watch mode, this version has a more intuitive UI in the browser.